### PR TITLE
README: the CoreOS way

### PR DESCRIPTION
**Pull request by bcwaldon**
This is an attempt at incorporating @polvi's original PR (#12) into master.

### CODE REVIEW

**Comment by philips** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -1,11 +1,26 @@
 # Kubernetes on CoreOS
 
-Follow the Kubernetes guides on the CoreOS website:
+This repo contains tooling and documention around deploying Kubernetes using CoreOS.
+Initial setup of a Kubernetes cluster is covered, but ongoing maintenance and updates of the cluster is not addressed.
```
Comment (aaronlevy) [inline README.md]: Nit: Don't think it's necessary to call out that not covering updates/maintenance.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+## The CoreOS Way
+
+When designing these guides and tools, the following considerations are made:
+
+* We always setup TLS
+* An individual node can reboot and the cluster will still function
+* Internal cluster DNS is available
+* Service accounts enabled
+* Use a cloud-provider if we can, for instance on AWS
+* Use Kubernetes for AdmissionControllers and other suggested guidelines
```
Comment (aaronlevy) [inline README.md]: Nit: Not really sure what this means. Maybe "Use **suggested** Kubernetes AdmissionControllers"?

**Comment by aaronlevy** [comment]
Couple nits but otherwise, lgtm

File: README.md (hunk 1 above)
Comment (bcwaldon) [inline README.md]: leaving it

File: README.md (hunk 2 above)
Comment (bcwaldon) [inline README.md]: good catch

**Pull request merged**